k8s/influxdb/values.yaml

```diff
@@ -0,0 +1,249 @@
+## influxdb image version
+## ref: https://hub.docker.com/r/library/influxdb/tags/
+image:
+  repository: "influxdb"
+  tag: "1.7.10-alpine"
```

Comment:
We need 1.8+.

Reply:
Need to create the `testground` database by default, and disable HTTP authentication (for now). Refer to the local healthchecks for more info on the env variables we set.

Reply:
> Need to create the testground database by default, and disable HTTP authentication (for now). Refer to the local healthchecks for more info on the env variables we set.

This is already done.

Reply:
Upgraded to 1.8 and works the same way.